Content.Shared/DoAfter/SharedDoAfterSystem.Update.cs

```diff
@@ -70,6 +70,18 @@ protected void Update(
                 continue;
             }
 
+            if (!doAfter.Completed && !doAfter.Cancelled && doAfter.Args.TargetEffect != null)
+            {
+                if (doAfter.LastEffectSpawnTime == null || time - doAfter.LastEffectSpawnTime.Value >= TimeSpan.FromSeconds(1))
+                {
+                    if (xformQuery.TryGetComponent(doAfter.Args.Target, out var targetXform))
+                    {
+                        SpawnAttachedTo(doAfter.Args.TargetEffect, targetXform.Coordinates);
```

Comment:
This needs an INetManager.IsServer check so it doesn't spam spawn clientside entities